src/QMCWaveFunctions/Jastrow/DiffTwoBodyJastrowOrbital.h

```diff
@@ -50,6 +50,8 @@ class DiffTwoBodyJastrowOrbital : public DiffWaveFunctionComponent
   std::map<std::string, FT*> J2Unique;
 
 public:
+  // return for testing
+  std::vector<FT*>& getFvector() { return F; }
```

Comment:
A more descriptive name would be nice - maybe getPairFunctions?

Reply:
Fixed!

Reply:
Fixed